cognite/powerops/custom_modules/power_model_v1/data_models/views/shop_result/1-interface.SHOPResult.view.yaml

```diff
@@ -23,56 +23,67 @@ filter:
 implements: []
 version: '{{version}}'
 properties:
-  scenario:
+  case:
     container:
       space: '{{powerops_models}}'
-      externalId: FunctionData
+      externalId: SHOPResult
       type: container
-    containerPropertyIdentifier: direct1
-    name: scenario
-    description: The Shop scenario that was used to produce this result
+    containerPropertyIdentifier: case
+    name: case
+    description: The case that was used to produce this result
     source:
       space: '{{powerops_models}}'
-      externalId: Scenario
+      externalId: Case
       version: '{{version}}'
       type: view
-  price_scenario:
+  outputTimeseries:
     container:
       space: '{{powerops_models}}'
-      externalId: FunctionData
+      externalId: SHOPResult
       type: container
-    containerPropertyIdentifier: direct2
-    name: price_scenario
-    description: The price scenario that was used to produce this result
-    source:
+    containerPropertyIdentifier: tsList1
+    name: tsList1
+    description: A general placeholder for all timeseries that ste, from a shop run
+  objectiveSequence:
+    container:
       space: '{{powerops_models}}'
-      externalId: PriceScenario
-      version: '{{version}}'
-      type: view
-  production:
+      externalId: SHOPResult
+      type: container
+    containerPropertyIdentifier: objectiveSequence
+    name: objectiveSequence
+    description: The sequence of the objective function
```

Comment:
I have another suggestion (or two) for how to represent the objective function results in the benchmarking data model - a sequence is really not a great fit for this - but we can change that later.